sdk/loadtestservice/test-resources.json

```diff
@@ -0,0 +1,85 @@
+{
```

Comment:
replace with real ARM template?

Reply:
@christothes I apologies, I am not able to understand this comment, can you please explain in more detail?

Reply:
It appears that this only defines a KeyVault resource. I expected to see an Azure Load Testing resource that would be the target for live tests